src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -58,13 +58,8 @@ module SoilBiogeochemCarbonFluxType
      real(r8), pointer :: lithr_col                                 (:)     ! (gC/m2/s) litter heterotrophic respiration: donor-pool based definition
      real(r8), pointer :: somhr_col                                 (:)     ! (gC/m2/s) soil organic matter heterotrophic res: donor-pool based definition
      real(r8), pointer :: soilc_change_col                          (:)     ! (gC/m2/s) FUN used soil C
-
-     ! fluxes to receive carbon inputs from FATES
```

Comment:
I think this is 3 litter types, with the first one being the sum of the last 3?

Reply:
I will give it a comment to be less confusing.  These variables are redundant, because they are exact copies of variables that exist in the interface. So we were allocating and copying over for no reason. In fact, we were copying twice, since there is the 3d and 2d versions of the same thing here. Instead, the routine UpdateCLitterFluxes() and UpdateNLitterFluxes() fill the cf_soil%decomp_cpools_sourcesink_col arrays directly from the FATES output structures: this%fates(ci)%bc_out(s)%litt_flux_lab_c_si(1:nlevdecomp), this%fates(ci)%bc_out(s)%litt_flux_cel_c_si(1:nlevdecomp), and this%fates(ci)%bc_out(s)%litt_flux_lig_c_si(1:nlevdecomp)

Reply:
the new variable is just there for mass balance checking purposes